btk/measure.py

```diff
@@ -91,12 +91,13 @@ def sep_measure(batch, idx):
     stamp_size = image.shape[-2]  # true for both 'NCHW' or 'NHWC' formats.
     coadd = np.mean(image, axis=0)
     bkg = sep.Background(coadd)
+    # Here the 1.5 value corresponds to a 1.5 sigma threshold for detection against noise.
     catalog, segmentation = sep.extract(coadd, 1.5, err=bkg.globalrms, segmentation_map=True)
     n_objects = len(catalog)
     segmentation_exp = np.zeros((n_objects, stamp_size, stamp_size), dtype=bool)
     deblended_images = np.zeros((n_objects, *image.shape), dtype=image.dtype)
     for i in range(n_objects):
-        segmentation_exp[i][np.where(segmentation == i + 1)] = True
+        segmentation_exp[i][segmentation == i + 1] = True
         seg_i = (segmentation == i + 1).reshape(1, stamp_size, stamp_size)
         deblended_images[i] = seg_i * image
```

Comment:
I suggest something even more straightforward
```suggestion
        seg_i = segmentation == i + 1
        segmentation_exp[i] = seg_i
        deblended_images[i] = image * seg_i[np.newaxis, ...]
```

Reply:
Thanks I like this approach @aboucaud